lib/sgtn-client/api/translation.rb

```diff
@@ -77,6 +80,7 @@ def self.getStrings(component, locale)
       private
 
       def self.getTranslation(component, key, locale)
+        locale = SgtnClient::LocaleUtil.get_best_locale(locale)
         items = get_cs(component, locale)
```

Comment:
If locale is source locale, return source directly.
```suggestion
        if locale == LocaleUtil.get_source_locale
          items = getSources(component, locale)
        else
          items = get_cs(component, locale)
        end
```

Reply:
get_best_locale will replace 'en' with SgtnClient::Config.configurations.default so the source bundle will be loaded.